src/app/api/dataset-flow.api.ts

```diff
@@ -211,6 +214,22 @@ export class DatasetFlowApi {
             );
     }
 
+    public getFlowById(params: { datasetId: string; flowId: string }): Observable<GetFlowByIdQuery> {
+        return this.datasetFlowByIdGQL
+            .watch(
+                { datasetId: params.datasetId, flowId: params.flowId },
+                {
+                    fetchPolicy: "no-cache",
+                    errorPolicy: "all",
+                },
```

Comment:
I think these are repeating, and we could declare constants for typical modes

Reply:
Fixed